levels/Level1/Mod4Recipes/TypingTutor.html

```diff
@@ -22,59 +22,62 @@
 <h1>Typing Tutor</h1>
 <br>
     <p class="normal">
-We are going to make a typing game.
+We are going to make a Typing Tutor .
         </p><br> 
  <p class="normal">
-     1. Create a frame for your typing game.<br>
+     1. Create a JFrame for your Typing Tutor.<br>
      <br>
      <img alt="" src="images/typingTutor2.png" style="width: 444.00px; height: 163.00px; margin-left: 100.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title="">
      
      
     <br>
-2. Create a member variable and initialize it as shown below:</p><br>
+2. Create a member variable in class and initialize it in the main method as shown below:</p><br>
```

Comment:
Since this is level 1 shouldn't they be initializing it in a setup method or similar non-static method? Otherwise they will have to make their member variable static and will not be able to add the keylistener.

Reply:
So this is about creating member variable for char  currentLetter for storing output of method generateRandomLetter(). 
But yes i should add instruction for making a setup method and creating all JComponents there .

Reply:
I made some changes to add a setup method.